tests/messages/test_pofile.py

```diff
@@ -429,6 +429,62 @@ def test_missing_plural_in_the_middle(self):
         self.assertEqual("", message.string[1])
         self.assertEqual("Vohs [text]", message.string[2])
 
+    def test_abort_invalid_po_file(self):
+        invalid_po = '''
+            msgctxt ""
+            "{\"checksum\": 2148532640, \"cxt\": \"collector_thankyou\", \"id\": "
+            "270005359}"
+            msgid ""
+            "Thank you very much for your time.\n"
+            "If you have any questions regarding this survey, please contact Tim "
+            "at andrea.raynorhentschel@dfo.ca."
```

Comment:
Please make absolutely sure there aren't any emails or names belonging to real people in the test data. (Consider this duplicated for all the names and emails here.)

Reply:
👍